packages/generator/templates/app/app/auth/mutations/changePassword.ts

```diff
@@ -12,7 +12,14 @@ export default resolver.pipe(
     const user = await db.user.findFirst({ where: { id: ctx.session.userId as number } })
     if (!user) throw new NotFoundError()
 
-    await authenticateUser(user.email, currentPassword)
+   try {
+      await authenticateUser(user.email, currentPassword)
+    } catch (error: any) {
+      if (error instanceof AuthenticationError) {
+      throw new Error("Invalid Password")
+      }
+      throw new Error(error)
```

Comment:
Since the `error` is already of the Error instance, we can just throw it:
```suggestion
      throw error
```